app/processing/statement/components/get-address.js

```diff
@@ -4,10 +4,10 @@ const getAddress = async (sbi, transaction) => {
   const organisation = await getOrganisation(sbi, transaction)
   return {
     line1: organisation.line1,
-    line2: organisation.line2,
+    line2: organisation.line2? organisation.line2: '',

```

Comment:
`default('')` in the schema means these should already be `''`

Reply:
Json object returns it as null when not explicitly declared.